docs/installation/steps/install-cuda.md

```diff
@@ -0,0 +1,33 @@
+# Install CUDA Toolkit:
+
+## What is CUDA?
+
+![CUDA](https://oerc.ox.ac.uk/media/12659/nvidia-cuda.jpg?anchor=center&mode=crop&width=950&height=460&rnd=133263876560000000)
+
+CUDA, which stands for Compute Unified Device Architecture, is a parallel computing platform and Application Programming Interface (API) developed by NVIDIA. CUDA enables developers to harness the immense computational power of NVIDIA GPUs (Graphics Processing Units) for a wide range of general-purpose computing tasks, including deep learning, scientific simulations, image processing, and more.
+
+## Why is CUDA important?
+
```

Comment:
@maryalexa91 I think instead of listing out the use cases of CUDA, we can mention the reason the we **need** this to be installed is that Tensorflow & XGBoost requires CUDA tookit in order to be able to make use of the GPU acceleration. Without it, they can only use CPU.

Reply:
I already added the reason for using CUDA